bftengine/src/bcstatetransfer/DBDataStore.cpp

```diff
@@ -59,6 +59,18 @@ void DBDataStore::load() {
   }
   loadResPages();
   loadPendingPages();
+
+  LOG_DEBUG(logger(), "MyReplicaId: "               << inmem_->getMyReplicaId());
```

Comment:
Nit: I think it's preferable to put all relevant info on a single log line rather than many. Either way there should probably be some context about why those values are being logged at the beginning of logging.

Reply:
It happens only once during startup.
In log4cplus the context gained through an appender - there's no reason to do it twice.
In our log, if we still plan to use it, we can add __PRETTY_FUNCTION__